--locale zh-CN doesn't work on Linux

```
daniel@drosen-ubuntu:~/foo$ tsc --locale zh-CN
foo.ts(1,1): error TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number' or an enum type.
daniel@drosen-ubuntu:~/foo$ tsc --locale zh-cn
foo.ts(1,1): error TS2362: The left-hand side of an arithmetic operation must be of type 'any', 'number' or an enum type.
```

Seems to work fine in Windows.

--- a/scripts/generateLocalizedDiagnosticMessages.ts
+++ b/scripts/generateLocalizedDiagnosticMessages.ts
@@ -65,10 +65,11 @@ function main(): void {
      * There are three exceptions, zh-CN, zh-TW and pt-BR.
      */
     function getPreferedLocaleName(localeName: string) {
+        localeName = localeName.toLowerCase();
         switch (localeName) {
-            case "zh-CN":
-            case "zh-TW":
-            case "pt-BR":
+            case "zh-cn":
+            case "zh-tw":
+            case "pt-br":
                 return localeName;
             default:
                 return localeName.split("-")[0];

